### PR TITLE
Feature/auth flow

### PR DESCRIPTION
**Pull request by skabanets**

### CODE REVIEW

File: env.example [hunk 1]
```diff
@@ -0,0 +1 @@
+VITE_API_BASE_URL=
```
Comment (AndriiKasian) [inline env.example:1]: - [ ] Why is this variable equal to nothing?

File: src/components/LoginForm/LoginForm.tsx [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+  formStyles,
+  deviderStyles,
+  inputStyles,
+  passwordIconStyles,
+  fieldWrapperStyles,
+  errorMessageStyles,
+} from "./LoginFormStyles";
+
+export const LoginForm = () => {
+  const [showPassword, setShowPassword] = useState<boolean>(false);
```
Comment (AndriiKasian) [inline src/components/LoginForm/LoginForm.tsx:30]: - [ ] move all props and actions that are related to the form, to the separate hook

File: src/services/api.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import axios from "axios";
```
Comment (AndriiKasian) [inline src/services/api.ts:1]: - [ ] let's use React Query library to make API calls

**Review by AndriiKasian — CHANGES_REQUESTED** [review]

File: env.example (hunk 1 above)
Comment (skabanets) [reply env.example:1]: @AndriiKasian The env.example file shows the environment variables we need, but it doesn’t include values. The empty VITE_API_BASE_URL= line helps developers know which variable they should add to their own .env file.

File: src/components/LoginForm/LoginForm.tsx (hunk 2 above)
Comment (skabanets) [reply src/components/LoginForm/LoginForm.tsx]: @AndriiKasian I've moved all the props and actions related to the form to a separate hook as suggested.

File: src/services/api.ts (hunk 3 above)
Comment (skabanets) [reply src/services/api.ts]: @AndriiKasian I've used RTK Query for making API calls and removed Axios from the project.

**Review by AndriiKasian — APPROVED** [review]

**Pull request merged**